src/main/webapp/index.html

```diff
@@ -54,59 +54,67 @@ <h1>TheGlobetrotter</h1>
 
     <!-- Main section -->
     <div id="map-section">
-      <div id="route-creation-section">
-        <textarea id="route-name" placeholder="Route name..."></textarea><br />
-        <ul id="places-table">
-          <div class="marker-setting">
-            <form id="marker-form">
-              <input
-                type="text"
-                id="marker-name"
-                name="marker-name"
-                placeholder="Marker name"
-              /><br />
-
-              <div class="setting-line">
-                <div class="time-text">Visit time</div>
-                <input
-                  type="text"
-                  id="visit-hour"
-                  name="visit-hour"
-                  value="-1"
-                /><br />
-                <div class="time-text">:</div>
+      <div id="route-content">
+        <div id="route-creation-section">
+          <textarea id="route-name" placeholder="Route name..."></textarea><br />
+          <ul id="places-table">
+            <div class="marker-setting">
+              <form id="marker-form">
                 <input
                   type="text"
-                  id="visit-minute"
-                  name="visit-minute"
-                  value="-1"
+                  id="marker-name"
+                  name="marker-name"
+                  placeholder="Marker name"
                 /><br />
-              </div>
 
-              <div class="setting-line">
-                <div class="time-text">Leave time</div>
-                <input
-                  type="text"
-                  id="leave-hour"
-                  name="leave-hour"
-                  value="-1"
-                /><br />
-                <div class="time-text">:</div>
-                <input
-                  type="text"
-                  id="leave-minute"
-                  name="leave-minute"
-                  value="-1"
-                /><br />
-              </div>
+                <div class="setting-line">
+                  <div class="time-text">Visit time</div>
+                  <input
+                    type="text"
+                    id="visit-hour"
+                    name="visit-hour"
+                    value="-1"
+                  /><br />
```

Comment:
Wrapping the input in a `<div>` tag is preferable to using `<br />` to move it to a new line. Line breaks are not CSS friendly, so in general using paragraph or div tags to moe something onto a new line make is easier to change layout and formatting in future.